packages/marketplace/src/components/hocs/error-boundary.tsx

```diff
@@ -40,7 +40,8 @@ export class ErrorBoundary extends React.Component<ErrorProps, ErrorState> {
       type: 'COMPONENT',
       message: errorMessages.DEFAULT_COMPONENT_ERROR,
     })
-    if (process.env.NODE_ENV === 'production') {
+    const isDevelopment = window.reapit.config.appEnv === 'development'
```

Comment:
So what is the app env for local development? I mean, sometimes we need to differentiate between local and development?